### PR TITLE
fix(VTextField): inherit outlined border radius

### LINKED ISSUE #17380: [Bug Report][3.2.4] rounded="lg" prop is not passed down to v-text-field when variant is set to outlined
### Environment
**Vuetify Version:** 3.2.4
**Vue Version:** 3.3.2
**Browsers:** Chrome 113.0.0.0, Google Chrome
**OS:** Mac OS 10.15.7, Windows

### Steps to reproduce
- Create a `v-text-field` component
- Change its variant to outlined
- add the rounded="lg" prop to the component

### Expected Behavior
Expected the border radius of the outline textfield to increase

### Actual Behavior
The border radius remains the same

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlV1tv2zYU/iunwrAkgCU7aQt0XhKk2OWpDwPWFgOqPlASbbOlSI2k7HiB//s+krrZSYcMTQBbOvf7Of70kGxb7sRqn32xyTIRdaONowcqDWeOf4w4OtDK6JrOOtqzXOWK3wfSUivrqEPQzTHj+UOuiNyG13wJoRVfsVa69/H9TIr1xp3RYeaJOpxdBhaij+/5vftdcFn1EKItM4Ipt6Q80a2TQvEqTwK3/6u4ssLtPbbU9QrGsULyCUGppTYe3RhRM7OfoiSz1qOMblXFq1Su8yQiD/4LH4eLXCWz5G3TZPAWsbp2vG4knL31FNfblDVNeAwvNROqe8P7izSllZAy3XDvNAlLLCqdkdIOL43RDaXpyLJNEVoHKdxEyps8mYjIE1rJVlQDQ68G0aVtavSOfsQ3fKYduGjDZQM1EsqEIsudE2pNbUNrIypy+4aTZHuEFRYJam3LpNyT5GzLqUpXkt8TQkq2BhwWuQ3YLb4YjDe8c6JmX0AT5USSqUvBKW/Yl9b66oBDJVeOG/jCUAxqApgw5S7EQvr8+SBcXoHe1nh6g4e6wsPrI4YxFn86pipmKri9htdwoCbr2tXq2Kxeh6fsQ203rIKpDUu9uq4sAPeFcaorJAvMqRNO8jFbyF+6C9lKCy0RZZR0unkFAe+8PdfzKds3hdq2OJZbO2/T7W8+UoRQG6SLG8VqjpQ3oNlpOOI0rbmDu8w4Xg26emnf9gFGTlU9EdtIHaJ5B3m1cFlj+Bap8wHyrj3J02flnWaVLw2Y5rhvhbXWVaiuAmUJnz78NSO70S1CVoCgRm9seYVi1eo0cRN7QnRXfmKcpGvwZtrctJTRjGByeAIQfasrLgHrQwqgZEUAfRhBhv/dCsOrWx/XUfEzTHtS7URvn7+J3j9GkO/TY6L/Y0kM/1rqgkn6oZ/ZmKq+gELFVMJipO3RyoXU5Vcq2tYh6FphFuC7736LKY9pm0G7Ff9wP+FpK/jOi7Aclbphas3/I1eFU09HYhn0AtSbl3UmZbZ+q6pf9Q61NSQ0+Fvoap9eon7Gbqt5Jdp6jFcsUj8thOrmJ6CvXoYhigl2Wi9+Ufgw93ti7FcY/lQ3AOPb4XFTDS0OS0+wPe4IHIBaTmcmIBiaEYLQWcXKrwUzw1IcSmfAdJuLqMZGFalflLR0ouYYy97vxWIx8dK2Zcmt7bkG1Q8PYWDRActv2Er9zoNai12yZKUTuACOZzyCNPWpW9tdvk7TpzSsGxOJyTblvSulCPXQO4cTY8WkRROOZEcR/EVqYI/CN03a9fxobUeCXngAYReEJHTLd0I27HT/Erf9kTi8ojNEg6nLXevRwz1l+Gp6RYULimg+x2aaDEMPiwdVD77xnOfB44ufO5YweOM0QWjtyDTM/sgVGSJq2BBTVCdvCG1Iz8ASkh/Jz8565QMt5vPu1OhJjkarpz75JXxDzO5VSecXN7fdZQcbKgwOXfN4M+yE2xBrBNYZa90mEnURybZMtt4NZ1oejCIf7fexus+9VBoORu/CwJAn6OI15py/f2LNr1ocOS/ypJMDSZ0HT2l5bEJwsMMeZnT1erGIcTrg83oea6GrC7fvdm4WL4vOxkLfpxGwpAX+r5p7MuuCnS9m4T9bXF3MAL/0iMs3j7A/XdCLWGTosKg76JtHhThYIxa12+DE1wqHa1Cddwj0PU7zaEzufwz49zzZONfY5XxeVgpsmC9iazLF3Vw19fwOZHPTKj9S0krXdy+zl9nVHJPaTcEZt3VaYHah9iBkOLeDGj/bn6Gqo4SGq+xV1NCBUqxH61U8Ej2Hvi03qeEY6sZfk8/z6IRt6tUJ6pFn/W8ExBvT9qs9CXXpx+uSPvVGfpfLQZiX9NkrPMySYCc0B4bk879jlWb4)

### Other comments
I am not sure if this was intentional for the outline variant only as it worked for the fill variant.

### PR DESCRIPTION
**Pull request by SonTT19**
## Description

fixes #17380

## Markup:

```vue
<template>
  <v-app>
    <v-container>
      <v-locale-provider rtl>
        <v-text-field v-model="msg" variant="outlined" rounded="lg" />
      </v-locale-provider>

      <v-text-field v-model="msg" variant="outlined" rounded="lg" />
    </v-container>
  </v-app>
</template>

<script setup>
  import { ref } from 'vue'

  const msg = ref('Hello World!')
</script>

```

### CODE REVIEW

**Comment by johnleider** [comment]
When you have a moment could you ping me on Discord? https://community.vuetifyjs.com

**Comment by johnleider** [comment]
@KaelWD can we use inline-border classes? i.e. **border-start-start-radius**.

**Comment by KaelWD** [comment]
No: https://caniuse.com/css-logical-props

File: packages/vuetify/src/components/VField/VField.sass [hunk 1]
```diff
@@ -362,10 +362,16 @@
       border-inline-start-width: var(--v-field-border-width)
 
       @include tools.ltr()
-        border-radius: $field-border-radius 0 0 $field-border-radius
```
Comment (KaelWD) [inline packages/vuetify/src/components/VField/VField.sass:365]: Was there a problem with `inherit 0 0 inherit`?

Comment (SonTT19) [reply packages/vuetify/src/components/VField/VField.sass:365]: @KaelWD Following to the document: https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius
```
Note: As with any shorthand property, individual sub-properties cannot inherit, such as in border-radius:0 0 inherit inherit, which would partially override existing definitions. Instead, the individual longhand properties have to be used.
```

Comment (KaelWD) [reply packages/vuetify/src/components/VField/VField.sass:365]: Well that's lame

**Review by KaelWD — APPROVED** [review]

**Pull request merged**